bpf/kmesh/workload/include/backend.h

```diff
@@ -28,7 +32,31 @@ static inline backend_value *map_lookup_backend(const backend_key *key)
 
 static inline int backend_manager(ctx_buff_t *ctx, backend_value *backend_v)
 {
+	int ret;
 	address_t target_addr;
+	__u32 *sk = (__u32 *)ctx->sk;
+	struct bpf_sock_tuple value_tuple= {0};
+
+	if (backend_v->waypoint_addr != 0 && backend_v->waypoint_port != 0) {
+		BPF_LOG(DEBUG, BACKEND, "find waypoint addr=[%u:%u]\n", backend_v->waypoint_addr,
+			backend_v->waypoint_port);
```

Comment:
In ambient, waypoint listens on 15008 by default. But in kmesh, waypoint should listen on 15019, maybe we could hardcode it in this release and make it configurable in the future.

Reply:
We currently hardcode it in the  kmesh-daemon code.

Reply:
this PR: https://github.com/kmesh-net/kmesh/pull/220/files